content/adventures/en.yaml

```diff
@@ -1005,6 +1005,40 @@ adventures:
                     {else}:
                         {print} 'Better luck next time..'
 
+    clear_command:
+        name: "{clear}"
+        description: clear command
+        default_save_name: clear_command
+        levels:
+            4:
+                start_code: |
+                    print '3'
+                    sleep
+                    clear
+                    print '2'
+                    sleep
+                    clear
+                    print '1'
+                    sleep
+                    clear
+                    print 'SURPRISE!'
+                story_text: |
+                    Time for a new command! With `{clear}` you can clear all the text form your output screen. This way you can prevent your screen getting too full of text.
+                    Mind! If you are using a `{clear}` command, please remember to use a `{sleep}` as well. Else Hedy will clear your screen without giving you the time to read it!
```

Comment:
```suggestion
                    Beware! If you are using a `{clear}` command, remember to use a `{sleep}` above it. Otherwise Hedy will clear your screen without giving you the time to read!
```